sinon/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Sinon 1.16.0
+// Type definitions for Sinon 2.1.0
 // Project: http://sinonjs.org/
-// Definitions by: William Sears <https://github.com/mrbigdog2u>
+// Definitions by: William Sears <https://github.com/mrbigdog2u>, Jonathan Little <https://github.com/rationull>
```

Comment:
Good point, I guess that's the main implication! I'm not even close to an authority on it and don't feel a need to approve future contributions. However I'm happy to be a reviewer to help out.